debian/changelog

```diff
@@ -1,4 +1,4 @@
-cloud-init (23.4.1-0ubuntu1~22.04.3) UNRELEASED; urgency=medium
+cloud-init (24.1.1-0ubuntu1~22.04.1) jammy; urgency=medium
```

Comment:
As with #5001:
```suggestion
cloud-init (24.1-0ubuntu1~22.04.1) jammy; urgency=medium
```

Reply:
fixed this as well as the commit message